pkgs/development/tools/misc/mkrom/default.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub, stdenv, cmake, asciidoc, libxslt }:
+
+
+stdenv.mkDerivation rec {
+  pname = "mkrom";
+
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "mkrom";
+    rev = version;
+    sha256 = "0smilzkpf4v4w5lbx4jaah8hr24110makxjy3j92qb404pxcb5r1";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ asciidoc libxslt.bin ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv bin/mkrom $out/bin
+  '';
```

Comment:
`cmake` has install stanzas. Did you try without `installPhase`? Removing this should also properly install the man pages.

Reply:
I tried removing `installPhase` and the build failed.